### ISSUE TITLE
Error connecting but the provided log file is empty

### ISSUE THREAD

**Issue by JAicewizard** [eduvpn/python-eduvpn-client#617]
# Installation

I used the AUR version of this client

# Version

`eduVPN GUI version: 4.4.0 with eduvpn-common version: 2.1.0`

# OS/Distribution

arch linux

# Logs

`~/.config/eduvpn/log` is empty, journalctl does provide useful information.

However, the GUI points tells me to look at `~/.config/eduvpn/log` which is empty. `~/.config/eduvpn/log` should contain at least some information, or more hints on where to look. Only when filing this bug report did I see I should check the sys logs

**Comment by JAicewizard** [comment]
I have found the issue, it was a kernel module not being loaded (I updated my system without restarting so that makes sense). The issue is that eduVPN does not point to `journalctl` when there is an issue like this, it points to its own logs which are empty.

A solution would be to either write a message to the log file indicating the issue lies elsewhere (e.g. `An unexpected error in NetworkManager occurred when connecting. For more information check journalctl .... `), or show a similar message in the GUI/TUI.

I can send the journalctl logs over the weekend, but I don't think they will provide much useful information.
